### PR TITLE
LEDs and new profiled PID values

### PR DESCRIPTION
**Pull request by zabzzz**

### CODE REVIEW

**Review by minhd-vu — APPROVED** [review]
if tested already go ahead and merge in, looks good!

**Comment by minhd-vu** [comment]
Let's also try to merge @croissant3's autos in whenever we can as well.

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -31,6 +31,7 @@ public void robotInit() {
     m_robotContainer = new RobotContainer();
     SmartDashboard.putNumber("Test Pivot", ManipulatorConstants.PIVOT_MAX);
     SmartDashboard.putNumber("Shooter Test", 2500);
+    RobotContainer.ledSubsystem.orange();
```
Comment (Shiv-Shah) [inline src/main/java/frc/robot/Robot.java]: Better funtion name

File: src/main/java/frc/robot/commands/manipulator/feeder/Rumble.java [hunk 2]
```diff
@@ -33,6 +33,7 @@ public void execute() {
       RobotContainer.getCoDriverController().getHID().setRumble(RumbleType.kRightRumble, ManipulatorConstants.RUMBLE_SETTING);
       RobotContainer.getCoDriverController().getHID().setRumble(RumbleType.kLeftRumble, ManipulatorConstants.RUMBLE_SETTING);
       RobotContainer.limeSubsystem.blinkLEDs();
+      RobotContainer.ledSubsystem.green();
```
Comment (Shiv-Shah) [inline src/main/java/frc/robot/commands/manipulator/feeder/Rumble.java]: turnGreen()

**Review by Shiv-Shah — CHANGES_REQUESTED** [review]

**Review by Shiv-Shah — APPROVED** [review]

**Pull request merged**